website/docs/docs/cloud/manage-access/auth0-migration.md

```diff
@@ -108,7 +108,7 @@ Below are steps to update. You must complete all of them to ensure uninterrupted
 
 <Lightbox src="/img/docs/dbt-cloud/access-control/redirect-URI.png" title="Enter new redirect URI"/>
 
-4. Navigate to the dbt Cloud environment and open the **Account Settings**. Click the **Single Sign-on** option from the left side menu and click the **Edit** option from the right side of the SSO pane. The **domain** field is the domain your organization uses to login to Azure AD. Toggle the **Enable New SSO Authentication** option and **Save**. _Once this option is enabled, it cannot be undone._
+4. Navigate to the dbt Cloud environment and open the **Account Settings**. Click the **Single Sign-on** option from the left side menu and click the **Edit** option from the right side of the SSO pane. The **domain** field is the domain your organization uses to login to Microsoft Entra ID. Toggle the **Enable New SSO Authentication** option and **Save**. _Once this option is enabled, it cannot be undone._
 
 :::warning Domain authorization
 
```

Comment:
below this, there's an imagine text that references Azure AD -- should we change that to Microsoft Entra ID?
![Screenshot 2024-03-21 at 12 50 31](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/ea844d6f-70e9-48a1-b4ee-4ca227fe5c51)